core/config/config.go

```diff
@@ -86,8 +92,8 @@ func Load(ctx context.Context, db pg.DB, rDB *raft.Service) (*Config, error) {
 				// If we got this far but failed to delete from PG, it's really NBD. Just
 				// log the failure and carry on.
 				log.Error(ctx, err, "failed to delete config from postgres")
-
 			}
+			err = migrateAccessTokens(ctx, db, rDB)
```

Comment:
This error strikes me as both 1) unlikely and 2) serious. So I vote that if it happens we should crash and get the customer to call us to help with the migration. They can continue running the old version in the meantime.